python/Ganga/GPIDev/Adapters/IBackend.py

```diff
@@ -475,10 +472,6 @@ def master_updateMonitoringInformation(jobs):
                     temp_block = []
 
                 for this_block in monitorable_blocks:
-
-                    if monitoring_component and not monitoring_component.isEnabled(False) or not monitoring_component:
```

Comment:
This is here to get out of the loop during shutdown, does it need to be removed?